adapters/taboola/taboola.go

```diff
@@ -288,3 +289,12 @@ func overrideBidRequestImp(originBidRequest *openrtb2.BidRequest, imp []openrtb2
 	bidRequestResult.Imp = imp
 	return &bidRequestResult
 }
+
+func resolveMacros(bid *openrtb2.Bid) {
+	if bid == nil {
+		return
+	}
```

Comment:
this function is not returning anything. So, `return` statement here sounds a bit odd. It might be better if we re-write this function as - 
```
func resolveMacros(bid *openrtb2.Bid) {
           if bid != nil {
           //set price, bid.NURL, bid.AdM here.
           }